src/status_im/contexts/profile/settings/screens/notifications/view.cljs

```diff
@@ -0,0 +1,102 @@
+(ns status-im.contexts.profile.settings.screens.notifications.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [react-native.platform :as platform]
+    [status-im.common.events-helper :as events-helper]
+    [status-im.config :as config]
+    [status-im.feature-flags :as ff]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn toggle-notifications-enabled
```

Comment:
I noticed there were a few comments on the code inside the legacy screen, so I thought it would be best to remove the old legacy screen and replace it with the updated version with the new design system components.

Please take a look at some of the fresh code when you have time 🙏 

cc @clauxx @ulisesmac @ilmotta